src/const.js

```diff
@@ -142,24 +153,15 @@ export const FLAGGED_FILE_MAGIC_NUMBERS = [
 // Based on the above, this is how deep we need to look into a file.
 export const FLAGGED_FILE_MAGIC_NUMBERS_LENGTH = 4;
 
-export const DEPRECATED_APIS = [
-  'app.getDetails',
-  'extension.onRequest',
-  'extension.onRequestExternal',
-  'extension.sendRequest',
-  'tabs.getAllInWindow',
-  'tabs.getSelected',
-  'tabs.onActiveChanged',
-  'tabs.onSelectionChanged',
-  'tabs.sendRequest',
-];
-
 // These are APIs that will cause problems when loaded temporarily
 // in about:debugging.
+// APIs listed here should be defined in https://mzl.la/31p4AMc
```

Comment:
at the MDN page linked here there are also a couple of other APIs (in particular: the native messaging API methods, `runtime.connectNative` and `runtime.sendNativeMessage`, and the `pkcs11` API namespace), should we add those here too?

Reply:
Ah yes, I wasn't quite sure if the pkcs11 namespace would simply work like that and wanted to put that into a separate issue.

Let me file a follow-up issue to take a deeper look at the docs and add the remaining APIs to the list if that works for you?

Reply:
sure, pkcs11 wasn't there and so it is definitely not a blocking issue. 
Looking into it as part of a follow-up issue works for me too.